packages/rum/src/viewTracker.ts

```diff
@@ -131,7 +131,11 @@ function trackMeasures(lifeCycle: LifeCycle, scheduleViewUpdate: () => void) {
         loadEventEnd: msToNs(navigationEntry.loadEventEnd),
       }
       scheduleViewUpdate()
-    } else if (entry.entryType === 'paint' && entry.name === 'first-contentful-paint') {
+    } else if (
+      entry.entryType === 'paint' &&
+      entry.name === 'first-contentful-paint' &&
+      document.visibilityState === 'visible'
```

Comment:
IMO, this check is not enough to prevent first contentful paint to be collected when page become visible
Have you tested it?

Reply:
Yes. 
I used the scenario described in the "First Contentful Paint in RUM" document.
That check prevented the `first_contentful_paint` from being added the the `measures` when the page is not visible.